### PR TITLE
feat(ticket rules): Turn on Ticket Rules

### PR DESCRIPTION
**Pull request by mgaeta**
This PR adds Ticket Rules as a feature for Jira and Azure Devops so that it can be used Business for AM1 plans only.
Screen Shot 2021-01-21 at 10 28 07 AM

### CODE REVIEW

**Comment by scefali** [comment]
@mgaeta I'm not seeing the description for ticket rules when I run this with Getsentry (but it appears with Sentry). Maybe the component we use limits how may items we can have in the list? Not sure.

Screen Shot 2021-01-13 at 3 05 56 PM

**Comment by scefali** [comment]
@mgaeta we probably want to also update the default value of the feature flag so on-prem users will get it: https://github.com/getsentry/sentry/blob/master/src/sentry/conf/server.py#L879

But that could also wait until a future PR (it doesn't need to be in this one and might be less risky if it's omitted).

File: tests/sentry/api/endpoints/test_project_rules_configuration.py [hunk 1]
```diff
@@ -93,11 +93,4 @@ def test_available_actions(self):
 
         action_ids = [action["id"] for action in response.data["actions"]]
         assert EMAIL_ACTION in action_ids
-        assert JIRA_ACTION not in action_ids
-
-    def test_ticket_rules_in_available_actions(self):
-        with self.feature("organizations:integrations-ticket-rules"):
```
Comment (scefali) [inline tests/sentry/api/endpoints/test_project_rules_configuration.py:99]: @mgaeta We should make sure that when `organizations:integrations-ticket-rules` is `False` that we don't render the Jira action.

**Comment by ceorourke** [comment]
@mgaeta since we decided on biz plan, this should be updated since it's listed as "for team billing plans and above"

**Comment by mgaeta** [comment]
@ceorourke I updated the description with a screenshot of the integration page.

File: tests/sentry/api/endpoints/test_project_rules_configuration.py (hunk 1 above)
Comment (mgaeta) [reply tests/sentry/api/endpoints/test_project_rules_configuration.py:99]: I added the test back and updated the function names.

**Review by scefali — APPROVED** [review]

**Comment by scefali** [comment]
> @mgaeta since we decided on biz plan, the plan features page should be updated since it's listed as "for team billing plans and above".
> 
> edit - err does it do that automatically with the plan chosen?

@ceorourke we shouldn't have any reference to billing plans in Sentry. All that should be in Getsentry.

**Comment by ceorourke** [comment]
@scefali I was referring to the screenshot, when you add the feature description it shows up depending on which plan it's on for availability

**Comment by scefali** [comment]
> @scefali I was referring to the screenshot, when you add the feature description it shows up depending on which plan it's on for availability

@ceorourke I see. Yea, not sure why a getsentry screenshot is in this PR...I guess Marcos can answer

**Pull request merged**